commcare_connect/templates/opportunity/invoice_detail.html

```diff
@@ -16,7 +16,18 @@
     {% crispy form %}
   </div>
 
-  <div class="mt-6 flex justify-between">
+
+  <div class="mt-6 flex justify-start gap-4">
+    {% if invoice_status == 'pending' %}
```

Comment:
I think this button should also be restricted to only NM users. My understanding is that only NM users should need to submit for approval. Otherwise the button to submit for approval will be shown for both NM and PM uses.
```suggestion
    {% if invoice_status == 'pending' and not request.is_opportunity_pm %}
```

Reply:
Great catch. 
Addressed in https://github.com/dimagi/commcare-connect/pull/887/commits/6fe7dc8aa9defd91abf56320730f6b31193561db